testsuite/microprofile-tcks/config/pom.xml

```diff
@@ -94,6 +94,13 @@
                <dependenciesToScan>
                   <dependency>org.eclipse.microprofile.config:microprofile-config-tck</dependency>
                </dependenciesToScan>
+
+               <properties>
+                  <property>
+                     <name>listener</name>
+                     <value>org.wildfly.swarm.microprofile.config.tck.TestNGAnnotationTransformer</value>
```

Comment:
@heiko-braun Why don't you add the exclusions to https://github.com/wildfly-swarm/wildfly-swarm/blob/master/testsuite/microprofile-tcks/config/src/test/tck-suite.xml ?

Reply:
None of the ways I've tried to add exclusion to `test-suite.xml` did work. It seems testng internally uses annotations to to categorize tests and these can be used for exclusion, but not class names or other direct references. This was the best way I could figure out.

Reply:
Hm, `test-suite.xml` should work (we're using it e.g. in CDI TCK; https://github.com/cdi-spec/cdi-tck/blob/master/impl/src/main/resources/tck-tests.xml#L28) and imho should be the preferred way of exclusion if using testng. I'll try it.